cmd/vcert/main.go

```diff
@@ -105,7 +105,7 @@ ACTIONS:
    getpolicy    To retrieve the certificate policy of a zone
    setpolicy    To apply a certificate policy specification to a zone
 
-   getcred      To obtain a new token for authentication
+   getcred      To obtain a new token for authentication on TPP or create a user account on VaaS
```

Comment:
To obtain a new TPP authentication token or register a VaaS user account

Reply:
it's done